pkg/plugins/platform/local.go

```diff
@@ -498,3 +498,8 @@ func (l *LocalApp) AddHostsEntry() error {
 	err = system.RunCommandPipe("sudo", hostnameArgs)
 	return err
 }
+
+// constructContainerName builds a container name given the type (web/db/dba) and the appName
+func constructContainerName(containerType string, appName string) string {
```

Comment:
In all places you are using it I believe you could also just use `app.FindContainerByType("web")` or `app.FindContainerByType("db")` which uses `FindContainerByLabels()` under the hood.